src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+
+    private int number;
+
+    public Ball() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new Random().nextInt(100);
+    }
```

Comment:
Move logic to class Lottery

```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```

Reply:
not fixed